### DISCUSSION TITLE
Form.Item normalize 对初始值无效

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#45941] [Q&A]
目前,只有手动输入的值才会执行, 但是初始值和手动输入最后都是需要提交的, 所以应该支持初始值, 即在提交时也需要调用一次

场景是` initialValues: {arr:[] } <Form.Item normalize={String}  <Select>`, Select 需要初始值为[], 所以我不能自己实现

**Comment by zombieJ** [comment]
可以 HOC 一下 FormItem，如果存在 `normalize` 就直接加一个 `initialValue` 属性上去。

`normalize` 不能自动执行，因为不知道子组件 onChange 会给什么类型。它也就没办法去模拟了。

**Reply by vaynevayne** [reply]
那换个思路,  添加一个normalize 对应的parse 方法, 这样 多选组件, initialValue写成 '', parse 成[], 这样的话,也就不需要 normalize在初始化时执行了,或者 select 组件本身添加一个也行
